tuna/export_db.py

```diff
@@ -386,25 +392,29 @@ def insert_perf_db_sqlite(session, cnx, perf_db_entry, ins_cfg_id):
   LOGGER.info("Inserting row in perf_db: %s", perf_db_dict)
 
 
-def get_query(dbt, args):
+def get_pdb_query(dbt, args):
   """Compose query to get perf_db rows based on filters from args"""
   with DbSession() as session:
-    query = session.query(dbt.perf_db_table, dbt.perf_config_table,
-                          dbt.config_table)\
-        .filter(dbt.perf_db_table.valid == 1)\
-        .filter(dbt.perf_db_table.session == dbt.session.id)\
-        .filter(dbt.perf_db_table.miopen_config == dbt.perf_config_table.id)\
-        .filter(dbt.perf_config_table.config == dbt.config_table.id)
+    query = session.query(dbt.find_db_table,
+                          dbt.config_table, dbt.tensor_table)\
+        .filter(dbt.find_db_table.valid == 1)\
+        .filter(dbt.find_db_table.kernel_time != -1)\
+        .filter(dbt.find_db_table.params != '')\
+        .filter(dbt.find_db_table.session == dbt.session.id)\
```

Comment:
please add this as the first filtering param, followed by valid then the rest

Reply:
makes sense